Package.swift

```diff
@@ -74,6 +78,12 @@ let package = Package(
             exclude: ["Info.plist"],
             publicHeadersPath: "Public"
         ),
+        .target(
+            name: "BraintreeDataCollector",
+            dependencies: ["BraintreeCore"],
```

Comment:
Similar to what we talked about yesterday with Magnes, maybe we should list KountDataCollector as a dependency here rather than in the BraintreeDataCollector library (line 28).

Reply:
Yeh - I'm not sure if it will provide any benefit. But, we can trial it with `PayPalDataCollector` and see if it no longer requires Drop-in to explicitly require PPRiskMagnes.